versions.mdx

```diff
@@ -0,0 +1,67 @@
+---
+{
+    "0.1.0": {
+        "firebase_core": "2.30.0",
+        "firebase_auth": "4.19.2",
+        "cloud_firestore": "4.17.0",
+        "firebase_database": "10.5.2",
+        "firebase_storage": "11.7.2",
+        "firebase_messaging": "14.8.2",
+        "firebase_crashlytics": "3.5.2",
+        "firebase_performance": "0.9.4+2",
+        "firebase_remote_config": "4.4.2",
+        "firebase_analytics": "10.10.2",
+        "firebase_in_app_messaging": "0.7.5+2",
+        "firebase_app_check": "0.2.2+2",
+        "firebase_ml_model_downloader": "0.2.5+2",
+        "native_sdk": {
+            "android": "32.8.0",
+            "ios": "10.24.0",
+            "web": "10.10.0",
+            "windows": "11.6.0"
+        },
+    }
+}
```

Comment:
We ought to put this at the bottom of the file. Also - this is still displayed on preview mode when we should probably hide it:
<img width="1410" alt="Screenshot 2024-04-18 at 10 13 28" src="https://github.com/firebase/flutterfire/assets/16018629/b95fa043-f905-4079-a646-d791ae72d783">

also - a trailing comma here which would break parsing I think:
```json
        "native_sdk": {
            "android": "32.8.0",
            "ios": "10.24.0",
            "web": "10.10.0",
            "windows": "11.6.0"
        },
```

Reply:
I'm quite keen on keeping it at the top since it's part of MDX spec as [Frontmatter](https://mdxjs.com/guides/frontmatter/), at the bottom it's just random things, as you can see if I add it to the bottom:
![Screenshot 2024-04-18 at 11 38 54](https://github.com/firebase/flutterfire/assets/3680002/cd9d820b-1809-47a7-9c01-4e2290d5d845)

Reply:
I've fixed the trailing comma